modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-publish.adoc

```diff
@@ -0,0 +1,103 @@
+include::_attributes.adoc[]
+endif::[]
+
+The Publish operation enables you to create a new Anypoint MQ message and send it to the specified destination: queue, FIFO queue, or message exchange. With it, you can configure both the content of the message and all the headers and properties you need.
+
+== Outgoing Message Structure
+
+Messages contain these parameters:
+
+* `body`
++
+Contains the body of the published Anypoint MQ message.
+* `properties`
++
+Represents a simple key/value map of Strings that are the message properties of the message.
+
+== Send a Message
+
+When used in its default form, the connector publishes the content found in the incoming message `payload` as the `body` of the Anypoint MQ message, serialized as a String, to the declared `destination`:
+
+image::anypoint-mq/3.x/amq-3x-publish-operation.png[Anypoint MQ Publish Operation]
+
+[source,xml]
+----
+<anypoint-mq:publish destination="invoiceQueue" config-ref="Anypoint_MQ_config"/>
+----
+
+This operation produces the following output:
+
+* A copy of the body sent as payload.
++
+For example, if you are sending a "Hello world" message, the resulting payload is the "Hello world" String.
+* The message's `messageId` in the message attributes.
+
+image::anypoint-mq/3.x/amq-3x-publish-attributes.png[Anypoint MQ Publish Operation Output Attributes]
+
+
+== Configure User Properties
+
+You can define properties for outgoing messages, for example, to provide compatibility with other messaging systems or to communicate the content-type of a message.
+
+Use the `properties` parameter of the Publish operation to configure these properties directly in the message. Define the `properties` parameter as a key/value map of Strings using DataWeave:
+
+image::anypoint-mq/3.x/amq-3x-publish-properties.png[Anypoint MQ Publish With Properties]
+
+[source,xml,linenums]
+----
+<anypoint-mq:publish config-ref="Anypoint_MQ_Config"
+    destination="invoiceQueue">
+  <anypoint-mq:properties><![CDATA[#[output application/java ---
+      {
+          AUTH_TYPE: 'jwt',
+          AUTH_TOKEN: attributes.queryParams.token
+        }]]]>
+      <anypoint-mq:properties>
+</anypoint-mq:publish>
+----
+
+=== Transform the Message Body
+
+If the payload is not in the correct format, you can transform it to the correct format.
+
+To transform a payload, declare an inline DataWeave transformation in the `body` parameter of the Publish operation:
+
+image::anypoint-mq/3.x/amq-3x-publish-transformed.png[Anypoint MQ Publish With Transformation]
+
+[source,xml,linenums]
+----
+<anypoint-mq:publish config-ref="Anypoint_MQ_Config"
+   destination="invoiceQueue">
+      <anypoint-mq:body ><![CDATA[#[output application/json ---
+      {
+        customer: payload.id,
+        items: payload.invoice.items
+      }]]]>
+      </anypoint-mq:body>
+</anypoint-mq:publish>
+----
+
+In this case, the result of the operation is the _transformed_ value as JSON.
+
+== Advanced Configurations
+
```

Comment:
GLOBALLY provide text between two contiguous headings. 
For container topics, you can list or summarize the subtopics, addressing the audience directly and telling them WHY they want or need to consume this content.